docs/identities/sign-in/actions.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: actions
+title: Customize login and registration flows with webhooks and actions
+sidebar_label: Actions and webhooks
```

Comment:
Why no `slug`?

Reply:
Did not know what good keywords would be

Reply:
So you `slug` as optional?

Reply:
Yes, optional!